src/web3mail/sendEmail.ts

```diff
@@ -145,6 +157,7 @@ export const sendEmail = async ({
         iexec_secrets: {
           1: emailSubjectId,
           2: emailContentId,
+          ...(vContentType ? { 3: contentTypeId } : {}),
```

Comment:
the options (requester secret 3) are pushed anyway, so I would always use it to leave le same on-chain trace regardless of the chosen options.
```suggestion
          { 3: optionsId },
```